db/migrations/20210129201213_single_name_field.sql

```diff
@@ -1,15 +1,29 @@
 -- migrate:up
 -- Add new single name field
-ALTER TABLE users ADD COLUMN name VARCHAR;
-COMMENT ON COLUMN users.name IS 'Name of user, may or may not be full name.';
+ALTER TABLE users ADD COLUMN full_name VARCHAR;
+COMMENT ON COLUMN users.full_name IS 'Name of user, may or may not be full name.';
```

Comment:
We should just name this field `name` in this case perhaps. I see no reason to change this.